components/bl-rating-reviews/src/views.js

```diff
@@ -0,0 +1,148 @@
+        </div>
+      </div>
+      <div className="data-container">
+        <div>
+          { reviewData.map(({ value }, i) => (
+            <div className="data-row" key={ i }>
+              <IconRow
+                Empty={ EmptyStar }
+                Filled={ FilledStar }
+                maxValue={ maxValue }
+                value={ maxValue - i }
+                color={ color }
+                size={ iconSize }
+              />
+              <span className="value">{ value }</span>
+            </div>
+          )) }
+        </div>
+        <div className="progress-bar-container">
+          { reviewData.map(({ value }, i) => (
+            <ProgressBar
+              key={ i }
+              eventHandlers={ eventHandlers }
+              maxValue={ totalReviews }
+              value={ value }
+              color={ color }
+            />
+          )) }
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const TwoColorReview = props => {
+  const { width, maxValue, color, iconSize, totalReviews, reviewData, eventHandlers } = props;
+
+  return (
+    <div className="twoColorReview" style={ { 'width': width } }>
+      <div className="data-container">
+        <div className="icons">
+          { reviewData.map((el, i) => (
+            <div className="icon-container" key={ i }>
+              <span className="number">{ maxValue - i }</span>
+              <span className="icon" style={ { 'height': iconSize, 'width': iconSize } }>
+                  <FilledStar/>
+                </span>
+            </div>
+          )) }
+        </div>
+        <div className="progress-bar-container">
+          { reviewData.map((el, i) => (
+            <ProgressBar
+              key={ i }
+              eventHandlers={ eventHandlers }
+              maxValue={ totalReviews }
+              value={ el.value }
+              color={ color }/>
+          )) }
+        </div>
+        <div className="values">
+          { reviewData.map((el, i) => (
+            <span key={ i }>{ el.value }</span>
+          )) }
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const MultiColorReview = props => {
+  const { width, average, maxValue, color, iconSize, totalReviews, reviewData, localizations, eventHandlers } = props;
+  return (
+    <div className="multiColorReview" style={ { 'width': width } }>
+      <div className="top-container">
+        <div className="average-value">{ average }</div>
+        <IconRow
+          Empty={ EmptyHeart }
+          Filled={ FilledHeart }
+          maxValue={ maxValue }
+          value={ average }
+          color={ color }
+          size={ iconSize }
+        />
+        <div className="total-reviews">{ totalReviews } { localizations.reviewsText }</div>
+      </div>
+      <div className="data-container">
+        <div className="numbers">
+          { reviewData.map((_, i) => (<span key={ i }>{ maxValue - i }</span>)) }
+        </div>
+        <div className="progress-bar-container">
```

Comment:
Find in code `progress-bar-container` it was written not once, but each time different code style. Also as far as it is duplicated, you could make it a component.